app/views/bets/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+  <% if current_user %>
+    <h2> Creating a new bet </h2>
+    <form action="<% create_bet_path %>" method="post" id="create-bet-form">
+      <div>
+        <input type="hidden" value="<%= form_authenticity_token %>" name="authenticity_token"/>
```

Comment:
You might want to stick to the rails form helpers. They provide a lot of begin the scenes help for you. 

Once example, is that if you try and edit an existing record. All of the records fields will be pre populated in the form fields
http://guides.rubyonrails.org/form_helpers.html